src/NzbDrone.Core.Test/MediaCoverTests/MediaCoverResizerFixture.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Core.MediaCover;
+using NzbDrone.Core.Test.Framework;
+using System.Linq;
+using System.IO;
+
+namespace NzbDrone.Core.Test.MediaCoverTests
+{
+    [TestFixture]
+    public class MediaCoverResizerFixture : CoreTest<MediaCoverResizer>
+    {
+        [Test]
+        public void should_resize_image()
+        {
+            var mainFile = Path.Combine(TempFolder, "logo.png");
+            var resizedFile = Path.Combine(TempFolder, "logo-170.png");
+
+            File.Copy(@"Files/1024.png", mainFile);
+
+            Subject.ResizeMediaCover(mainFile, resizedFile, 170, null);
+
+            var fileInfo = new FileInfo(resizedFile);
+            fileInfo.Exists.Should().BeTrue();
+            fileInfo.Length.Should().BeInRange(1000, 30000);
```

Comment:
Not without adding System.Drawing as reference and I wanted to avoid that. But I could make a hash of my local version and check it that way.